apps/nameguard.io/src/components/organisms/ContactUsForm.tsx

```diff
@@ -177,15 +178,15 @@ export const ContactUsForm = ({ title }: ContactUsFormProps) => {
           <p className="text-sm text-gray-500 mt-2 text-center">
             We have received your message and will get back to you soon.
           </p>
-          <button
+
+          <Button
```

Comment:
@eduramme Hey it seems something broke somewhere? I'm unable to submit this form now: https://nameguard-e7a2f4s1n-namehash.vercel.app/contact

I keep getting this error, even though my internet is still working fine.

![CleanShot 2024-09-03 at 17 31 49](https://github.com/user-attachments/assets/6aab0cd4-43ad-427d-a5e1-e4693d8040d4)